lib/ruby2d/text.rb

```diff
@@ -11,6 +11,10 @@ def initialize(x=0, y=0, text="Hello World!", size=20, font=nil, c='white', z=0)
       
       # if File.exists? font
         @font = font
+      end
+      
+      # if File.exists? font
+        # @font = font
       # else
```

Comment:
@KillaPL Just rebased this PR against master. What did you want to do here? Something like `@font = Font.new(font)`? Not sure how you imagined the Font interface.